### PR TITLE
Resolves #1393

### LINKED ISSUE #1393: Editorial: most instances of "States" and Properties" should be changed to "Attribute(s)"
I noticed in the diff of an unrelated PR that:
Line 13003 (currently) of the spec includes `<p>This <a>property</a> is marked on the members`…

According to spec style guide and terminology, the spec should always use the term "attribute" when referring to DOM content attributes, regardless of whether it is an ARIA `state` or ARIA `property`. We only use the terms `state` or `property` when 1) the difference is critical to understand in the moment, or 2) we're referring to DOM properties such as `element.ariaLabel`

From [6.1 Clarification of States versus Properties](https://w3c.github.io/aria/#statevsprop): 

> Because the distinction between states and properties is of little consequence to most web content authors, this specification refers to both "states" and "properties" simply as "attributes" whenever possible.

This probably means an editorial review of the spec is in order... Among other changes, all of the labels in the Characteristics tables (Inherited States and Props, Required States and Props, etc) could be changed (Inherited Attrs, Required Attrs, etc.) It's possible even some of the section headings (5.2.2–5.2.5) should also change.

I think making this change broadly may help reduced complexity of reading the spec, but I'm not

### PR DESCRIPTION
**Pull request by cookiecrook**
***
<a href="https://pr-preview.s3.amazonaws.com/w3c/aria/pull/1395.html" title="Last updated on Feb 5, 2021, 2:47 AM UTC (ece1167)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/aria/1395/02fee23...ece1167.html" title="Last updated on Feb 5, 2021, 2:47 AM UTC (ece1167)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -317,9 +317,9 @@ <h2><abbr title="Accessible Rich Internet Applications">WAI-ARIA</abbr> Roles</h
...
+		<h2><abbr title="Accessible Rich Internet Applications">WAI-ARIA</abbr> States and Properties (Attributes)</h2>
+		<p><abbr title="Accessible Rich Internet Applications">WAI-ARIA</abbr> provides a collection of accessibility <a>attributes</a> (as content <a>attributes</a>) which are used to support platform <a>accessibility <abbr title="Application Programing Interfaces">APIs</abbr></a> on various operating system platforms. <a>Assistive technologies</a> may access this information through an exposed <a>user agent</a> DOM or through a mapping to the platform accessibility <abbr title="Application Programing Interfaces">API</abbr>. When combined with <a>roles</a>, the user agent can supply the assistive technologies with user interface information to convey to the user at any time. Changes in attributes will result in a notification to assistive technologies, which could alert the user that a change has occurred.</p>
```
Comment (cookiecrook) [inline index.html:321]:
```suggestion
		<p><abbr title="Accessible Rich Internet Applications">WAI-ARIA</abbr> provides a collection of accessibility <a>states</a> and <a>properties</a> (as content <a>attributes</a>) which are used to support platform <a>accessibility <abbr title="Application Programing Interfaces">APIs</abbr></a> on various operating system platforms. <a>Assistive technologies</a> may access this information through an exposed <a>user agent</a> DOM or through a mapping to the platform accessibility <abbr title="Application Programing Interfaces">API</abbr>. When combined with <a>roles</a>, the user agent can supply the assistive technologies with user interface information to convey to the user at any time. Changes in attributes will result in a notification to assistive technologies, which could alert the user that a change has occurred.</p>
```

File: index.html [hunk 2]
```diff
@@ -10137,29 +10137,29 @@ <h5>Presentational Roles Conflict Resolution</h5>
...
 		</section>
 		<section id="propcharacteristic_usedinrole">
 			<h3>Used in Roles</h3>
-			<p>Advisory information about <a href="#role_definitions">roles</a> that use this <a>state</a> or <a>property</a>. This information is provided to help understand the appropriate usage of the state or property. Use of a given state or property is not defined when used on roles other than those listed.</p>
+			<p>Advisory information about <a href="#role_definitions">roles</a> that use this <a>attribute</a>. This information is provided to help understand the appropriate usage of the attribute. Use of a given attribute is not defined when used on roles other than those listed.</p>
 		</section>
 		<section id="propcharacteristic_inheritsintoroles">
 			<h3>Inherits into Roles</h3>
-			<p>Advisory information about <a href="#role_definitions">roles</a> that inherit the <a>state</a> or <a>property</a> from an ancestor role.</p>
+			<p>Advisory information about <a href="#role_definitions">roles</a> that inherit the <a>attribute</a> from an ancestor role.</p>
```
Comment (cookiecrook) [inline index.html:10158]:
```suggestion
			<p>Advisory information about <a href="#role_definitions">roles</a> that inherit the <a>state</a> or <a>property</a> from an ancestor role.</p>
```

File: index.html [hunk 3]
```diff
@@ -11648,7 +11648,7 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
 			<sdef>aria-hidden</sdef>
 			<div class="state-description">
 				<p>Indicates whether the <a>element</a> is exposed to an accessibility API. See related <sref>aria-disabled</sref>.</p>
-				<p>User agents determine an element's <a>hidden</a> status based on whether it is rendered, and the rendering is usually controlled by CSS. For example, an element whose <code>display</code> property is set to <code>none</code> is not rendered. An element is considered <a>hidden</a> if it, or any of its ancestors are not rendered or have their <code>aria-hidden</code> attribute value set to <code>true</code>.</p>
+				<p>User agents determine an element's <a>hidden</a> status based on whether it is rendered, and the rendering is usually controlled by CSS. For example, an element whose CSS <code>display</code> property is set to <code>none</code> is not rendered. An element is considered <a>hidden</a> if it, or any of its ancestors are not rendered or have their <code>aria-hidden</code> attribute value set to <code>true</code>.</p>
```
Comment (cookiecrook) [inline index.html:11651]: Added "CSS" here to help disambiguate one of the ~6 different uses of the term "property" in the spec.

File: index.html [hunk 4]
```diff
@@ -13304,15 +13304,15 @@ <h2>Including Elements in the Accessibility Tree</h2>
...
             </li>
           </ul>
 	</section>
 </section>
 <section class="normative" id="host_languages">
 	<h1>Implementation in Host Languages</h1>
-	<p>The <a>roles</a>, <a>state</a>, and <a>properties</a> defined in this specification do not form a complete web language or format. They are intended to be used in the context of a host language. This section discusses how host languages are to implement <abbr title="Accessible Rich Internet Applications">WAI-ARIA</abbr>, to ensure that the markup specified here will integrate smoothly and effectively with the host language markup.</p>
+	<p>The <a>roles</a> and <a>state</a>, and <a>properties</a> defined in this specification do not form a complete web language or format. They are intended to be used in the context of a host language. This section discusses how host languages are to implement <abbr title="Accessible Rich Internet Applications">WAI-ARIA</abbr>, to ensure that the markup specified here will integrate smoothly and effectively with the host language markup.</p>
```
Comment (cookiecrook) [inline index.html:13315]:
```suggestion
	<p>The <a>roles</a>, <a>states</a>, and <a>properties</a> defined in this specification do not form a complete web language or format. They are intended to be used in the context of a host language. This section discusses how host languages are to implement <abbr title="Accessible Rich Internet Applications">WAI-ARIA</abbr>, to ensure that the markup specified here will integrate smoothly and effectively with the host language markup.</p>
```

Comment (cookiecrook) [reply index.html:13315]: Correcting what appears to be a botched partial revert that I missed before committing.

**Comment by cookiecrook** [comment]
Quite disturbed to see there is a conflict with this giant diff.

**Comment by carmacleod** [comment]
> Quite disturbed to see there is a conflict with this giant diff.

Unfortunately, it seems that the parent of your commit is an old commit from September.
Near as I can tell, there have been 18 things merged since then, so resolving conflicts might be painful.

Your initial commit: https://github.com/w3c/aria/commit/6e1abfc1fdb30a41932533e531339ec0f6057cff

image

Your commit's parent: https://github.com/w3c/aria/commit/02fee233c1eb2981d337c539c922aba017a8ed84

image


If I was any good at git, maybe I could give you a magic git command to change your initial commit's parent.
But I don't have a clue how to do that, or even if it's possible. 😢 
@jnurthen Are you a git expert?

**Comment by cookiecrook** [comment]
I may be able to rebase. Not sure why this new branch came off an old head.

**Comment by carmacleod** [comment]
In general (just scrolling through the changes quickly), I really like the direction.
I'll wait for it to be based on the latest before reviewing thoroughly, though.

**Comment by carmacleod** [comment]
git rebase --onto <new-parent> <old-parent>
git rebase source-commit --onto target-branch

**Comment by carmacleod** [comment]
Caveat emptor: I use an IDE for git, so I should not be giving googled git command line suggestions! 😂 

I don't know if this will work:

```
git rebase --onto <new-parent> <old-parent>

i.e. 
$ git checkout main
$ git pull
$ git rebase --onto main 02fee23
```

Or maybe this?

```
$ git checkout main
$ git pull
$ git checkout issue-1393
$ git rebase main
$ git checkout main
$ git merge issue-1393
```

**Comment by cookiecrook** [comment]
Ah..... I branched from the old `master` branch, not the new `main` branch. That would do it. 😣

**Comment by cookiecrook** [comment]
For future reference, anyone could either delete their local copy and reclone, or run the same default branch name change commands... I hadn't, so my local branch was still telling me it was up to date with origin/master, even though origin/master didn't exist anymore... Frustrating.

**Comment by carmacleod** [comment]
Glad you figured it out!

I see that @jnurthen just updated the README with the necessary magic incantation: https://github.com/w3c/aria/blob/main/README.md#the-default-branch-has-been-renamed

Thanks so much @cookiecrook for saving the rest of us some pain! ❤️

**Comment by cookiecrook** [comment]
Closing.... I will start a new PR. It will be time consuming, but less risky than attempting to merge 6 months of changes.

**Pull request closed without merging**